packages/integrations/src/opnsense/opnsense-types.ts

```diff
@@ -0,0 +1,95 @@
+import { z } from "zod";
+
+// API documentation : https://docs.opnsense.org/development/api.html#core-api
+
+export const opnsenseSystemSummarySchema = z.object({
+  name: z.string(),
+  versions: z.array(z.string()),
+});
```

Comment:
We currently only use the versions, so I would suggest to remove the `name` property